src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml.cs

```diff
@@ -35,46 +36,37 @@ public IndexPageModel(ReviewManager manager, UserPreferenceCache preferenceCache
             int CurrentPage, int? PreviousPage, int? NextPage) PagedResults { get; set; }
 
         public async Task OnGetAsync(
-            List<string> search = null, List<string> languages=null, List<string> state =null,
-            List<string> status =null, List<string> type =null, int pageNo=1, int pageSize=_defaultPageSize, string sortField=_defaultSortField)
+            IEnumerable<string> search = null, IEnumerable<string> languages=null, IEnumerable<string> state =null,
+            IEnumerable<string> status =null, IEnumerable<string> type = null, int pageNo=1, int pageSize=_defaultPageSize, string sortField=_defaultSortField)
         {
-            if (search.Count == 0 && languages.Count == 0 && state.Count == 0 && status.Count == 0 && type.Count == 0)
+            if (!search.Any() && !languages.Any() && !state.Any() && !status.Any() && !type.Any())
```

Comment:
won't this fail is any of these args has default null value?

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.any?view=net-6.0